generator/src/main/java/com/scottlogic/deg/generator/StandardGenerationEngine.java

```diff
@@ -3,24 +3,30 @@
 import com.google.inject.Inject;
 import com.scottlogic.deg.generator.decisiontree.DecisionTreeCollection;
 import com.scottlogic.deg.generator.decisiontree.DecisionTreeFactory;
-import com.scottlogic.deg.generator.generation.DataGenerator;
-import com.scottlogic.deg.generator.generation.GenerationConfig;
+import com.scottlogic.deg.generator.generation.*;
 import com.scottlogic.deg.generator.outputs.GeneratedObject;
 import com.scottlogic.deg.generator.outputs.targets.OutputTarget;
 
 import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
 import java.util.stream.Stream;
 
 public class StandardGenerationEngine implements GenerationEngine {
     private final DecisionTreeFactory decisionTreeGenerator;
+    private ReductiveDataGeneratorMonitor monitor;
     private final DataGenerator dataGenerator;
+    public static int counter;
+    private SimpleDateFormat simpleDateFormat = new SimpleDateFormat("hh:mm:ss");
```

Comment:
Presentational detail shouldn't be present at this layer, it should be in the 'reporter' only